packages/renderer/src/components/EditorPage/component.tsx

```diff
@@ -38,6 +38,7 @@ export function EditorPage() {
     updateScene,
     loadingPreview,
     loadingPublish,
+    isInstalling,
```

Comment:
this `isInstalling` does not reflect the status of the npm install, but the Creators Hub install, so this is always going to be `false` at this point (it's only true the first time you open the CH or after an update, while you see the "Installing Decentraland Creator Hub vX.Y.Z") . We currently dont track in the app state the status of the npm installs.